src/dbnode/storage/shard.go

```diff
@@ -1878,6 +1878,29 @@ func (s *dbShard) Bootstrap(
 		s.setFlushStateColdVersion(at, info.VolumeIndex)
 	}
 
+	// May be nil depending on the caching policy.
+	if retrieverMgr := s.opts.DatabaseBlockRetrieverManager(); retrieverMgr!= nil {
+		// Attempt to call CacheShardIndices now that all the shards are bootstrapped. Cannot call
```

Comment:
nit: Update from "now that all the shards are bootstrapped" to "now that this shard is finishing bootstrapping".